src/components/revenue-graph/RevenueGraph.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+  <div class="revenue-graph">
+    <div v-for="month in revenue" :key="month.month" class="wrapper">
+      <div class="bar" :style="{height:barHeight(month.revenue)+'%'}"></div>
+      <div :class="['month', month.month === currentMonth() ? 'current' : '']">{{month.month}}</div>
+    </div>
+  </div>
+</template>
+
+<script>
+// import Foo from 'bar/foo'
+
+export default {
+  name: "revenue-graph",
+  mixins: [],
+  props: {
+    revenue: { type: Array },
+    currency: { type: String, default: "EUR" },
+    multiplication: { type: Number, default: 1 }
+  },
+  data() {
+    return {};
+  },
+  computed: {},
+  methods: {
+    currentMonth() {
+      let months = [
```

Comment:
Here is better to user momentjs, see [here](https://momentjs.com/docs/#/displaying/)

Reply:
importing moment, I have error on reload 
<img width="656" alt="Screen Shot 2019-09-13 at 2 13 41 PM" src="https://user-images.githubusercontent.com/41395780/64861675-d4e40b80-d630-11e9-93fd-52ef88a6fc4f.png">

Reply:
@IgorZ07 Can you push your actual code? I will pull it to see what's happened

Reply:
Pushed as requested